datahub-graphql-core/src/main/resources/entity.graphql

```diff
@@ -9059,3 +9080,26 @@ input BatchUpdateSoftDeletedInput {
     """
     deleted: Boolean!
 }
+
+enum UserSetting {
+    """
+    Show simplified homepage
+    """
+    SHOW_SIMPLIFIED_HOMEPAGE
+}
+
+
+"""
+Input for updating a user setting
+"""
+input UpdateUserSettingsInput {
```

Comment:
or maybe this should be updateUserSettingInput

Btw this diverges from how we do global settings, where we allow you to update a full blob at one time!